test/match-test.js

```diff
@@ -623,6 +623,70 @@ describe("sinonMatch", function () {
             assert(sinonMatch.isMatcher(array));
             assert.equals(array.toString(), "typeOf(\"array\")");
         });
+
+        describe("array.deepEquals", function () {
+            it("has a .deepEquals matcher", function () {
+                var deepEquals = sinonMatch.array.deepEquals([1, 2, 3]);
+
+                assert(sinonMatch.isMatcher(deepEquals));
+                assert.equals(deepEquals.toString(), "deepEquals([1,2,3])");
+            });
+
+            it("matches arrays with the exact same elements", function () {
+                var deepEquals = sinonMatch.array.deepEquals([1, 2, 3]);
+                assert(deepEquals.test([1, 2, 3]));
+                assert(!deepEquals.test([1, 2]));
```

Comment:
The negation here is easy to overlook. Could you change these to `assert.isFalse` throughout?